src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,85 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+/**
+ * Represents struct a user.
+ *
+ * @author Daniel De La Pava
+ */
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+
+    @NotNull
+    private String username;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private LocalDate birthDate;
+
+    /**
+     * Represents the foreign association between user and book
+     */
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY, mappedBy = "user")
```

Comment:
The kind of relationship is many to many, because a user can have many books and a book can be owned by many users :)